app/views/request/_share_by_link.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if can? :share, info_request %>
```

Comment:
Given this partial is included in `app/views/request/_sidebar.html.erb` and I can see the share by link section on a public request, I'm not sure this check is actually working:

![Screenshot 2020-04-09 at 13 30 27](https://user-images.githubusercontent.com/282788/78895236-605a3980-7a66-11ea-9cb0-d8ad233c3027.png)

Reply:
The cause of this has been fixed in #5612